tools/generate-docs.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Generate docs!
+go run github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs
+
+# Generate subcategories
+# Follow https://github.com/hashicorp/terraform-plugin-docs/issues/156 to see if anything is merged upstream. If so, we should use that.
+SCRIPTPATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+subcategories="$(cat ${SCRIPTPATH}/subcategories.json)"
+
+echo -e "\nAdding subcategories to docs..."
+for f in $(find ./docs/ -name "*.md"); do
+    f=${f#"./docs/"}
+    f="${f%.*}"
+    echo "  - ${f}"
+
+    subcategory="$(echo ${subcategories} | jq -r ".[\"${f}\"]")"
```

Comment:
There's no need to cat the contents of the JSON file in line 9 and the pipe it here, you can also do:
```suggestion
    subcategory=$(jq -r --arg cat "${f}" '.[$cat]' "${SCRIPTPATH}/subcategories.json")
```

Also, easier to read what's going on with `jq` 😸